apps/docker/docker-cmd.ts

```diff
@@ -3,7 +3,7 @@ import * as ink from "https://deno.land/x/ink/mod.ts";
 import {
   Input,
   Select,
-} from "https://deno.land/x/cliffy@v0.18.2/prompt/mod.ts";
+} from "https://deno.land/x/cliffy@v0.25.2/prompt/mod.ts";
```

Comment:
is there a way to centralise the list of dependencies and their versions? so one can update them all at once